### PR TITLE
[FRONT-360] Node-specific metrics & inertia dragging

### PR DESCRIPTION
**Pull request by tumppi**
Node-specific metrics are now hooked up. Also added inertia to map dragging.

### CODE REVIEW

File: src/components/Map/Map.stories.tsx [hunk 1]
```diff
@@ -110,14 +110,14 @@ export const ZoomControls: Story = (args) => {
   const onZoomIn = useCallback(() => {
     setViewport((prev) => ({
       ...prev,
-      zoom: prev.zoom + 1,
+      zoom: (prev.zoom || 0) + 1,
```
Comment (juhah) [inline src/components/Map/Map.stories.tsx:113]: Would this need `Math.max`... or does it matter 🤔

File: src/components/Map/index.tsx [hunk 2]
```diff
@@ -88,6 +91,18 @@ export const Map = ({
       getCursor={getCursor}
       ref={mapRef}
       onClick={onMapClick}
+      dragPan={{
```
Comment (juhah) [inline src/components/Map/index.tsx:94]: 👍

**Review by juhah — APPROVED** [review]
Great 👍 The `all` option doesn't seem to work, probably because that option doesn't have a case in `getResendOptionsForInterval`... not sure correct resend option there?

**Comment by juhah** [comment]
Added one related issue: https://linear.app/streamr/issue/FRONT-399/node-stat-graph-mb-s-hover-value

**Comment by tumppi** [comment]
Good point with `all` option. Added a case for that and it now resends all events from `day` stream beginning at the start of year 2021.

**Pull request merged**